sdks/typescript/featurehub-javascript-client-sdk/app/local_client_context.ts

```diff
@@ -0,0 +1,22 @@
+import { BaseClientContext } from "./context_impl";
```

Comment:
am I missing something, are there now two copies of this class?

Reply:
Sorry about that. I'm pretty sure I had removed but maybe it got back after my struggle rebasing. BTW, is the unittest for `LocalClientContext` in the right place (`client_context_spec.ts`)?

Should I drop this PR and start a new one fresh so that we can have a single commit?

Reply:
Keep the LocalCientContext in client_context.ts and then yes, its in the right place (the test). And yes, it would be nicer to have just a single commit :-)

Reply:
Sorry... I did't follow: would you like me to move `LocalCientContext` back to `client_context.ts`? I thought you wanted it out of that file due to separation of concerns.

Reply:
I did - but I said if we do that, we should also move the Server* and Client* ones out as well, otherwise people would get confused. For your commit, I think I'm being too much of a purist so leave it where you had it :-) Keep it easy.